Doc/whatsnew/3.9.rst

```diff
@@ -70,6 +70,17 @@ Summary -- Release highlights
 New Features
 ============
 
+Dictionary Merge & Update Operators
+-----------------------------------
+
+Merge (``|``) and update (``|=``) operators have been added to the built-in
+:class:`dict` class.
+
+.. XXX https://www.python.org/dev/peps/pep-0584/
```

Comment:
I'm not sure why you would want to keep a full link to the PEP in a comment. Do other sections do this?

Reply:
I misunderstood your prior review comment!